content/guides/development.md

```diff
@@ -5,182 +5,247 @@ contributors:
 
-webpack-dev-server can do many more things such as proxying requests to your backend server. For more configuration options, see the [**devServer documentation**](/configuration/dev-server).
 
+### Using webpack-dev-server
 
-### webpack-dev-middleware
+The `webpack-dev-server` provides you with a simple web server and the ability to use live reloading. Let's set it up:
 
-webpack-dev-middleware works for connect-based middleware stacks. This can be useful if you already have a Node.js server or if you want to have full control over the server.
-
-The middleware will cause webpack to compile files in-memory. When a compilation is running, it will delay the request to a file until the compilation is done.
-
-W> This is intended for advanced users. webpack-dev-server is much easier to use.
+``` bash
+npm install --save-dev webpack-dev-server
+```
 
-Start with installing the dependencies from npm:
+Change your config file to tell the dev server where to look for files:
+
+__webpack.config.js__
+
+``` diff
+  const path = require('path');
+  const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+  module.exports = {
+    entry: {
+      app: './src/index.js',
+      print: './src/print.js'
+    },
+    devtool: 'inline-source-map',
++   devServer: {
++     contentBase: './dist'
++   },
+    plugins: [
+      new HtmlWebpackPlugin({
+        title: 'Development'
+      })
+    ],
+    output: {
+      filename: '[name].bundle.js',
+      path: path.resolve(__dirname, 'dist')
+    }
+  };
+```
 
-```bash
-npm install --save-dev express webpack-dev-middleware
+This tells the dev server to serve our files from the `dist` folder (where our files get generated) and serve them on `localhost:8080`.
+
+Let's add a script to easily run the dev server as well:
+
+__package.json__
+
+``` diff
+  {
+    "name": "asset-management",
+    "version": "1.0.0",
+    "description": "",
+    "main": "webpack.config.js",
+    "scripts": {
+      "test": "echo \"Error: no test specified\" && exit 1",
+      "watch": "webpack --progress --watch",
++     "server": "webpack-dev-server --open"
+      "build": "webpack"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "devDependencies": {
+      "css-loader": "^0.28.4",
+      "csv-loader": "^2.1.1",
+      "file-loader": "^0.11.2",
+      "html-webpack-plugin": "^2.29.0",
+      "style-loader": "^0.18.2",
+      "webpack": "^3.0.0",
+      "xml-loader": "^1.2.1"
+    }
+  }
 ```
 
-After installing, you can use the middleware like this:
+Now we can run `npm run server` from the command line and we will see our browser automatically loading up a new page (usually `localhost:8080`). If you now change any of the source files and save them, the web server will automatically reload after the code has been compiled. Give it a try!
```

Comment:
Interesting, it would consistently fail on 8080 for me because there's my nginx proxy we use for docker.

Might be my setup though, so I took it out of the doc regardless :)

Thanks for explaining!